src/py/flwr/simulation/ray_transport/ray_client_proxy.py

```diff
@@ -115,6 +117,133 @@ def reconnect(
         return common.DisconnectRes(reason="")  # Nothing to do here (yet)
 
 
+class RayClientProxyForActorPool(ClientProxy):
```

Comment:
How about:
```suggestion
class RayActorClientProxy(ClientProxy):
```
We could also rename the legacy one to `RayTaskClientProxy`.